helix-core/src/test/java/org/apache/helix/integration/task/TestStuckTaskQuota.java

```diff
@@ -63,6 +63,9 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
+    System.out.println("AfterClass: " + testClassName + " of TestStuckTaskQuota called.");
```

Comment:
Same here. You can use methods to get the names here.

Reply:
changed to `    System.out.println("AfterClass: " + TestHelper.getTestClassName() + ":" + TestHelper.getTestMethodName()  + "invoked.");
`